packages/headless/src/features/search/legacy/search-actions-thunk-processor.ts

```diff
@@ -0,0 +1,364 @@
+import {AnyAction} from '@reduxjs/toolkit';
```

Comment:
Should we add legacy in this file's name ? This could get confusing with two files with same name so close to each other.

Reply:
That's why I put them in a /legacy folder, every import of it requires to go 'thru' /legacy because of this
(So, tldr, nope, already in the path so is ok)